GraphAlgorithms/Comparers/VertexThenEdge/VertexThenEdgeComparer.cs

```diff
@@ -33,6 +33,7 @@ public VertexThenEdgeSize GetSize(List<int> vertices)
         {
             foreach (var u in vertices)
             {
```

Comment:
When calculating the clique size I dont think we should counld edges to itself. If someone disagrees let me know